src/app/dashboard/Appbar.jsx

```diff
@@ -33,6 +33,9 @@ const useStyles = makeStyles((theme) => ({
   menuButton: {
     marginLeft: "4px",
   },
+  toolBarTitle: {
+    marginLeft: "1rem",
```

Comment:
can we use theme.spacing for as much as possible ?

Reply:
I've updated that entry and other changes I've made to use theme.spacing instead of rem or px. Let me know if those changes look good!